### ISSUE TITLE
Notify & JAWS popup navigation

### ISSUE THREAD

**Issue by joe-replin** [adaptlearning/adapt_framework#3176]
### Subject of the issue/enhancement/features
 If the user opens one of the hotgraphic/hotgrid items, all text in it is read out as expected. However, if they then move to the next/back quicknav arrows and hit Enter, the next/previous display text is shown and the heading is highlighted, but JAWS does not read the heading out immediately - instead the user has to move down to the body text and then back up to the header.

This only happens if the user is navigating with JAWS' virtual cursor (e.g. using the up/down arrow keys). If they move to the next/back arrow with the Tab key and then press Enter, the header next is read out as expected. I'm not sure if this is expected as the user is navigating through pop-ups during this time. Would changing this to read out each notify pop-up adversely affect the focus state for JAWS?

### Your environment
* v5.13.0
* Firefox 89.0.1
* Windows 
* Seen in JAWS 2020 & JAWS 2021

**Comment by oliverfoster** [comment]
Hi Joe,

Does this happen if you use Jaws and Chrome / Edge / IE11 or NVDA and Firefox?

**Comment by joe-replin** [comment]
Hey Ollie, here's my testing results. It looks like Firefox & any combination of NVDA behaves a bit differently upon opening the notify pop-up the first time versus every other browser/screen-reader combo.
Chrome & JAWS 2020 - Just the heading is read out upon opening the notify popup. Additional navigation is needed to read the entire popup. Further selecting the prev/next buttons within the popup only read out the title within the popup.
Chromium Edge & JAWS 2020 - Just the heading is read out upon opening the notify popup. Additional navigation is needed to read the entire popup. Further selecting the prev/next buttons within the popup only read out the title within the popup.
IE & JAWS 2020 - Just the heading is read out upon opening the notify popup. Additional navigation is needed to read the entire popup. Further selecting the prev/next buttons within the popup only read out the title within the popup.
Firefox & JAWS 2020 - **The entire notify popup is read out upon triggering the notify popup**. Further selecting the prev/next buttons within the popup only read out the title within the popup.
Firefox & NVDA - **The entire notify popup is read out upon triggering the notify popup**. Further selecting the prev/next buttons within the popup only read out the title within the popup.
Chrome & NVDA - **The entire notify popup is read out upon triggering the notify popup**. Further selecting the prev/next buttons within the popup only read out the title within the popup.
IE & NVDA - **The entire notify popup is read out upon triggering the notify popup**. Further selecting the prev/next buttons within the popup only read out the title within the popup.

**Comment by oliverfoster** [comment]
duplicate of https://github.com/adaptlearning/adapt-contrib-core/issues/89
